src/samples/Icons/index.html

```diff
@@ -1,57 +1,66 @@
-﻿<!DOCTYPE html>
+﻿<!-- Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information. -->
+
+<!doctype html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
+    <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Fabric Icons</title>
 
     <!-- Fabric core and Components CSS -->
     <link rel="stylesheet" type="text/css" href="../../css/fabric.css">
-    <link rel="stylesheet" type="text/css" href="../../css/fabric.components.css">    
-    
+    <link rel="stylesheet" type="text/css" href="../../css/fabric.components.css">
+
     <style type="text/css">
+        html,
         body {
-          margin: auto 20px;
+            margin: 0;
+            padding: 0;
         }
-
+        
         h1 {
-          padding: 8px;
-        }
-
-        .docs-IconsSection ul {
+            margin: 0;
+            padding: 10px;
             margin-bottom: 20px;
+            padding-left: 25px;
+        }
+        
+        .docs-Intro,
+        .docs-Example, 
+        .docs-IconsSection {
+            margin: 3vw;
```

Comment:
Not sure about the top and bottom margin for this. The spacing between the intro and the content below seems off.